frontend/app/templates/create-badges.hbs

```diff
@@ -1,8 +1,12 @@
 <div class="ui stackable grid">
-    <div class="right floated right aligned two wide column">
-        <button {{action 'togglePreview'}} class="tiny ui basic orange button" data-tooltip="Toggle Preview" data-position="left center">
-            <i class="exchange icon"></i>
-        </button>
+    <div class="right floated right aligned two wide column" style="margin-right:50px;">
+        
+
+<label class="switch" {{action 'togglePreview'}} onclick="document.getElementById('toggle').checked=!document.getElementById('toggle').checked" id="switch"   data-tooltip="Toggle Preview" data-position="left center" >
```

Comment:
`onclick` is already being triggered when it is clicked in the `{{action 'togglePreview'}}` I suppose.
Can you please see to it and make changes ?